packages/react-instantsearch/src/core/InstantSearch.test.js

```diff
@@ -76,7 +70,7 @@ describe('InstantSearch', () => {
           <div />
         </InstantSearch>
       );
-    }).toThrowError('You must provide a `onStateChange` prop alongside the `state`, `createURL` props on <InstantSearch>');
```

Comment:
Side note: I do like to always denote any API entry with backticks to ease reading anywhere :p